packages/lit-element/package.json

```diff
@@ -4,6 +4,7 @@
   "main": "index.js",
   "scripts": {
     "build": "npx rollup -c",
+    "copy": "echo \"No copy command installed\"",
```

Comment:
in cases where there is a rollup build, we don't need to run copy so we just add a dummy command to be run instead.